mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/domain/dto/UtkastDto.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.mulighetsrommet.api.domain.dto
+
+import kotlinx.serialization.Serializable
+import no.nav.mulighetsrommet.api.domain.dbo.UtkastDbo
+import no.nav.mulighetsrommet.api.domain.dbo.Utkasttype
+import no.nav.mulighetsrommet.domain.serializers.LocalDateTimeSerializer
+import no.nav.mulighetsrommet.domain.serializers.UUIDSerializer
+import java.time.LocalDateTime
+import java.util.*
+
+@Serializable
+data class UtkastDto(
+    @Serializable(with = UUIDSerializer::class)
+    val id: UUID,
+    val bruker: String,
+    val utkastData: String,
```

Comment:
Kan kanskje være `JsonElement`, burde ikke bli store forskjellen i serialisering/deserialisering?

Reply:
Har vel ikke så veldig mye å si om det er det ene eller det andre :thinking:

Reply:
Ja, jeg trenger litt bistand for korrekt serialisering/deserialisering. Jeg ble ikke helt klok på hva jeg skulle bruke, så jeg prøvde meg frem med bare string først.